packages/cfd/src/Components/props.types.ts

```diff
@@ -68,6 +68,68 @@ export type TTradingPlatformAvailableAccount = {
     sub_account_type: string;
 };
 
+export type TCardFlipStatus = {
+    svg: boolean;
+    bvi: boolean;
+    labuan: boolean;
+    vanuatu: boolean;
+    maltainvest: boolean;
+};
+
+export type TClickableDescription = {
+    type: 'text' | 'link';
+    text: string;
+};
+
+export type TJurisdictionCardSectionTitleIndicators = {
+    type: 'displayText' | 'displayIcons';
+    display_text?: string;
+    display_text_skin_color?: string;
+    display_icons?: Array<string>;
+};
+
+export type TJurisdictionCardSection = {
+    key: string;
+    title: string;
+    title_indicators?: TJurisdictionCardSectionTitleIndicators;
+    description?: string;
+    clickable_description?: Array<TClickableDescription>;
+};
+
+export type TJurisdictionCardItemVerification = Array<
+    'document_number' | 'selfie' | 'identity_document' | 'name_and_address'
+>;
+
+export type TJurisdictionCardItems = {
+    header: string;
+    over_header?: string;
+    synthetic_contents: Array<TJurisdictionCardSection>;
+    financial_contents: Array<TJurisdictionCardSection>;
+    is_over_header_available: boolean;
+    synthetic_verification_docs?: TJurisdictionCardItemVerification;
+    financial_verification_docs?: TJurisdictionCardItemVerification;
+};
+
+export type TJurisdictionCardBackRequiredDocsItem = {
```

Comment:
can we have a better name for this? something like TJurisdictionVerificationItem